client/src/components/graph/util.ts

```diff
@@ -127,3 +144,58 @@ export function createModelTF() {
   mat3.translate(m, m, [-0.5, -0.5]);
   return m;
 }
+
+export function getSpatialPrefixUrl(s3URI: string) {
+  const url = getSpatialUrl(s3URI);
+
+  return `${url}spatial_files/`;
+}
+
+export function getSpatialTileSources(s3URI: string) {
+  const url = getSpatialUrl(s3URI);
+
+  return `${url}spatial.dzi`;
+}
+
+function getSpatialUrl(s3URI: string) {
+  const datasetVersionId = getDatasetVersionId(s3URI);
+
+  const { hostname } = window.location;
+
+  // TODO(thuang): Take rdev into account
```

Comment:
NOTE: this logic currently doesn't take rdev into account. It's not working now anyway until we switch to Argus